frontend/test/metabase/scenarios/joins/reproductions/18512-cannot-join-two-saved-questions-with-same-implicit-explicit-grouped-field.cy.spec.js

```diff
@@ -30,10 +28,9 @@ describe("issue 18512", () => {
 
     cy.findByText("18512#1").click();
     cy.icon("join_left_outer").click();
-    cy.wait("@schema");
 
     popover().within(() => {
-      cy.findByTextEnsureVisible("Sample Database").click();
+      cy.findByTextEnsureVisible("Sample Database").click({ force: true });
```

Comment:
Why do we need to force the click if we're previously making sure this string is visible?

Reply:
Not sue TBH, but click sometimes was not working as expected, but with the force is OK

Reply:
What exactly was happening when it didn't work?
Do you maybe have some screenshot?

I mean, there's no harm in approving and merging this. Don't get me wrong.
But it's better if we first understand what was going on first.

Reply:
![Screen Shot 2022-05-04 at 16 56 06](https://user-images.githubusercontent.com/17742979/166815923-8395ac3e-f9e2-4cd2-8163-2800e83990e4.png)


![Screen Shot 2022-05-04 at 16 57 17](https://user-images.githubusercontent.com/17742979/166815934-a23f4743-2ac8-4900-a129-ea5cd58c429c.png)

Reply:
Hmmm... this seems like a popover placement/rendering thing.
Notice how it was still higher in the failed screenshot. So my guess is that Cypress finds "Sample Database" string and gets its coordinates before popover "settles" in the final position. When it tries to click on it, there's nothing there anymore.

But if that is the case, it feels very strange that force click would solve that problem. 🤔